projects/js-packages/scan/package.json

```
@@ -0,0 +1,68 @@
+{
+	"private": false,
+	"name": "@automattic/jetpack-scan",
+	"version": "0.1.0-alpha",
+	"description": "A JS client for consuming Jetpack Scan services",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/scan/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Scan"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/scan"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "pnpm run clean && pnpm run compile-ts",
+		"clean": "rm -rf build/",
+		"compile-ts": "tsc --pretty",
+		"test": "jest"
+	},
+	"type": "module",
+	"devDependencies": {
+		"@automattic/jetpack-webpack-config": "workspace:*",
+		"@babel/core": "7.23.5",
+		"@babel/plugin-transform-react-jsx": "7.23.4",
+		"@jest/globals": "29.7.0",
+		"@storybook/addon-actions": "7.4.6",
+		"@storybook/blocks": "7.4.6",
+		"@storybook/react": "7.4.6",
+		"@testing-library/react": "13.4.0",
```

The rest of the monorepo is already on 14.1.2
```suggestion
		"@testing-library/react": "14.1.2",
```